processor/processorhelper/obsreport.go

```diff
@@ -40,3 +41,17 @@ func (or *obsReport) recordInOut(ctx context.Context, incoming, outgoing int) {
 	or.telemetryBuilder.ProcessorIncomingItems.Add(ctx, int64(incoming), or.otelAttrs)
 	or.telemetryBuilder.ProcessorOutgoingItems.Add(ctx, int64(outgoing), or.otelAttrs)
 }
+
+func (or *obsReport) recordDuration(ctx context.Context, ps processorStart) {
+	duration := time.Since(ps.Time)
+	durationSecs := float64(duration.Microseconds()) / 1e6 // Convert to seconds
+	or.telemetryBuilder.ProcessorDuration.Record(ctx, float64(durationSecs), or.otelAttrs)
```

Comment:
```suggestion
	or.telemetryBuilder.ProcessorDuration.Record(ctx, duration.Seconds(), or.otelAttrs)
```

Your code will truncate to microsecond precision - is that intentional? Any reason to not use the Seconds method? If so can we document it please?